stream/stream_lavf.c

```diff
@@ -238,6 +238,96 @@ void mp_setup_av_network_options(AVDictionary **dict, const char *target_fmt,
     talloc_free(temp);
 }
 
+#define PROTO(...)(const char *[]){__VA_ARGS__, NULL}
+
+#define ADD_PROTOS(name, protos) \
```

Comment:
you can inline it, it is used only once now. Also break on first match.

Reply:
Ended up just removing it altogether.